optimize/components/userguide/process-analysis/report-analysis/measures.md

```diff
@@ -15,17 +15,17 @@ Reports with multiple aggregations that have a [second "Group by"](./define-repo
 ## User task duration time
 
 :::note
-The following information regarding idle versus work is currently applicable only to Camunda 7.
+The following information regarding unassigned versus assigned is currently applicable only to Camunda 7.
 :::
 
 In user task duration reports, you have the opportunity to select which part of the user task's lifecycle you want to see in the report:
 
-- Idle: View how long each user task was considered idle (not claimed by an assignee/user) during its execution.
-- Work: View how long each user task was considered to be worked on by assignees/users (claimed by an assignee/user) during its execution.
+- Unassigned: View how long each user task was considered assigned (not claimed by an assignee/user) during its execution.
+- Assigned: View how long each user task was considered to be assigned to assignees/users (claimed by an assignee/user) during its execution.
 - Total: View how long each user task took to complete.
 
 It is possible to display and compare multiple user task duration times in the same report. Reports with multiple user task duration times that have a [second "Group by"](./define-reports.md#reports-with-a-second-group-by-option) can only be visualized as table.
 
 :::note
-User tasks which have been completed yet have no claim operations are evaluated as follows: if the user task was canceled, the task is considered to have been idle whereas user tasks which were completed are considered to have been worked on programmatically or via a custom UI, meaning the time between start and end is considered work time.
+User tasks which have been completed yet have no claim operations are evaluated as follows: if the user task was canceled, the task is considered to have been unassigned whereas user tasks which were completed are considered to have been assigned programmatically or via a custom UI, meaning the time between start and end is considered assigned time.
```

Comment:
@toco-cam this is actually an interesting question: with the rename, should we be changing the logic described in this note?
Currently it works as described above, but if we rename idle/work time in the UI should these durations always be assumed unassigned no matter if the task was cancelled or completed?

Reply:
I don't understand this paragraph. I extracted the following out of it:

- Completed user tasks (Have not claimed operation) are evaluated as follows:
   - Canceled -> Considered "IDLE"
   - Completed -> Considered "Work-time"

Do we want to describe how we handle "Canceled" tasks in the reports?

Reply:
> Completed user tasks (Have not claimed operation) are evaluated as follows:
> Canceled -> Considered "IDLE"
> Completed -> Considered "Work-time"

Yes thats correct as of now. Since the rename of "idle" to "unassigned" and "work" to "assigned" time though this makes less sense, hence the question. We could for example count both scenarios as "unassigned" time since they are defacto unassigned. If we just rename these and leave the logic as is, userTasks that were completed without an assign operation will have this time counted as "assigned" which could be misleading.